lib/compiler/src/beam_trim.erl

```diff
@@ -212,12 +222,8 @@ expand_recipe({Layout,Trim,Moves}, FrameSize) ->
     end.
 
 create_map(Trim, []) ->
```

Comment:
Why remap through a fun, shouldn't it be enough to pass `Trim` and the move map to `remap`?